bindings/python/FloatingBaseEstimators/src/FloatingBaseEstimators.cpp

```diff
@@ -38,7 +38,24 @@ void CreateKinDynComputationsDescriptor(pybind11::module& module)
     py::class_<KinDynComputationsDescriptor>(module, "KinDynComputationsDescriptor")
         .def(py::init())
         .def_readwrite("kindyn", &KinDynComputationsDescriptor::kindyn)
-        .def("is_valid", &KinDynComputationsDescriptor::isValid);
+        .def("is_valid", &KinDynComputationsDescriptor::isValid)
+        .def("get_nr_of_dofs", &KinDynComputationsDescriptor::getNrOfDegreesOfFreedom)
+        .def("get_joint_pos", &KinDynComputationsDescriptor::getJointPos,
+              py::arg("q"))
+        .def("set_joint_pos",&KinDynComputationsDescriptor::setJointPos,
+              py::arg("s"))
+        .def("get_robot_state",&KinDynComputationsDescriptor::getRobotState,
+              py::arg("world_T_base"),
+              py::arg("s"),
+              py::arg("base_velocity"),
+              py::arg("s_dot"),
+              py::arg("world_gravity"))
```

Comment:
In order to be more "pythonic," I would avoid using mutable references for returning values. [Here](https://pybind11.readthedocs.io/en/stable/faq.html#limitations-involving-reference-arguments) you can find further details.
I think you can use `py::dict` to return a dictionary of different objects in python, otherwise you can return a tuple

Reply:
Could be addressed by https://github.com/dic-iit/bipedal-locomotion-framework/pull/303#discussion_r631066529

Reply:
Addressed via https://github.com/dic-iit/bipedal-locomotion-framework/pull/303/commits/3d3653036c2a534ff2126f9bf924755c991ccb13